components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -393,7 +393,7 @@ const ContributorProfileHeader = ({
 const AddToListDropdown = ({ username }: { username: string }) => {
   const [selectListOpen, setSelectListOpen] = useState(false);
   const [selectedList, setSelectedList] = useState<OptionKeys[]>([]);
-  const { data } = useFetchAllLists();
+  const { data } = useFetchAllLists({});
```

Comment:
refactor (blocking): passing an empty object doesn't make sense, it all the properies are optional, you can default to an empty object in the hook.